sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamEnumerableConverter.java

```diff
@@ -129,6 +140,11 @@ public static PipelineOptions createPipelineOptions(Map<String, String> map) {
     return options;
   }
 
+  static List<Row> toRowList(PipelineOptions options, BeamRelNode node) {
+    // TODO: support limit.
```

Comment:
nit: You should probably add the other cases here and throw an exception.

Reply:
@apilloud Thanks! Comment addressed.